spec/linter-eslint-spec.js

```diff
@@ -0,0 +1,68 @@
+'use babel';
+
+describe('The eslint provider for Linter', () => {
+  const {spawnWorker} = require('../lib/helpers')
+  const worker = spawnWorker()
+  const lint = require('../lib/main').provideLinter.call(worker).lint;
+
+  beforeEach(() => {
+    waitsForPromise(() => {
+      return atom.packages.activatePackage('language-javascript').then(() =>
+        atom.workspace.open(__dirname + '/files/good.js')
+      )
+    });
+  });
+
+  describe('checks bad.js and', () => {
+    let editor = null;
+    beforeEach(() => {
+      waitsForPromise(() => {
+        atom.config.set('linter-eslint.disableEslintIgnore', true);
+        return atom.workspace.open(__dirname + '/files/bad.js').then(openEditor => {
+          editor = openEditor;
+        });
+      });
+    });
+
+    it('finds at least one message', () => {
+      return lint(editor).then(messages => {
+        expect(messages.length).toEqual(1);
```

Comment:
Should it be at least one or exactly one? The assertion doesn't match the description.